### PR TITLE
ZA2 - Chandre De Wet - Github Module -  Fixed Linus Answers that had Errors

### PR DESCRIPTION
**Pull request by chandredewet**
# Your Details

- Your Name: Chandre De Wet
- Your City: South Africa
- Your Slack Name: Chandre De Wet

# Homework Details

- Module: Github
- Week: 1

-----
[View rendered homework.md](https://github.com/chandredewet/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,37 +44,37 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-When you send a file over the internet
+When you send a request to the owner of the repo to make changes
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It lets you see all changes made to the project.
```
Comment (AdvocateM) [inline homework.md:77]: shows a list of all the commits made to a repository

File: homework.md [hunk 2]
```diff
@@ -44,37 +44,37 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-You delete it
+You make your own copy of another repo on your own Github profile.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You download the repo to your pc.
```
Comment (AdvocateM) [inline homework.md:65]: what is i don't use pc? 

i think you should have generalized this by saying local machine / your local storage

**Review by AdvocateM — COMMENTED** [review]
otherwise you killed it

File: homework.md [hunk 3]
```diff
@@ -110,4 +110,4 @@ To record what changes you made to the code
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+The GitHub Desktop Cheatsheet.
```
[the remark this answers is not in the document]
Comment (chandredewet) [reply homework.md:113]: Hi, I was just referring to the cheatsheet we have - which is teh GitHub Desktop Cheatsheet - https://syllabus.codeyourfuture.io/git/desktop/cheatsheet

File: homework.md (hunk 1 above)
Comment (chandredewet) [reply homework.md:77]: Hi I think I was answering in laymans terms - as in a repository is a project and commits is the changes made, But I really need to get better at using the technical terms so thank you for pointing that out.

File: homework.md (hunk 2 above)
Comment (zackniyokwizera) [reply homework.md:65]: Well done @chandredewet  and @AdvocateM
